config/webpack.config.prod.js

```diff
@@ -170,30 +170,19 @@ module.exports = {
               sourceMaps: false
             }
           },
+          sharedConfig.tsLoader,
+          // Plain CSS file loading, primarily used to load from `node_modules` provided stylesheets
+          // e.g. mapbox, slick carousel, tippy, etc
           {
-            test: /draft-js.*\.css$/,
-            use: [
-              'style-loader',
-              'css-loader'
-            ]
-          },
-          {
-            test: /slick-carousel.*\.css$/,
-            use: [
-              'style-loader',
-              'css-loader'
-            ]
-          },
-          {
-            test: /mapbox-gl.*\.css$/,
+            test: /\.css$/,
```

Comment:
This allows general import of `.css` files, probably coming form something in `node_modules` as was the case with `mapbox-gl` and `slick-corousel`, and now `prosemirror`/TipTap. They will of course, appropriately, not be PostCSS or SCSS processed. I have tested fairly throughly and it's working. There were no `.css` files which were being processed through the `scss|sass` loader previously.

Reply:
👍